src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -0,0 +1,255 @@
+        climbPiston = map.getClimbPiston();
+        lidar = map.getLidar();
+        leftEncoder = map.getLeftEncoder();
+        rightEncoder = map.getRightEncoder();
+        gyro = map.getGyro();
+    }
+    // TODO put numbers here
+
+    double gyroCorrection;
+
+    PIDSource gyroSource = new PIDSource() {
+
+        @Override
+        public void setPIDSourceType(PIDSourceType pidSource) {
+
+        }
+
+        @Override
+        public double pidGet() {
+            return gyro.getAngle();
+        }
+
+        @Override
+        public PIDSourceType getPIDSourceType() {
+            return null;
+        }
+    };
+
+    PIDController gyroDrivePID = new PIDController(.01, .0009, 0.0, 0.0, gyroSource, (double value) -> {
+        gyroCorrection = value;
+    });
+
+    double sandstormSpeed = .2;
+
+    @Override
+    public void initDefaultCommand() {
+
+        // Set the default command for a subsystem here.
+        // setDefaultCommand(new MySpecialCommand());
+    }
+
+    public Command driveNormal() {
+        // The command is named "Sample Command" and requires this subsystem.
+        return new Command("driveNormal", this) {
+            @Override
+            protected void initialize() {
+                // Called just before this Command runs the first time
+            }
+
+            @Override
+            protected void execute() {
+                // Called repeatedly when this Command is scheduled to run
+                drive.arcadeDrive(
+                        Robot.driveController.getTriggerAxis(Hand.kRight)
+                                - Robot.driveController.getTriggerAxis(Hand.kLeft),
+                        Robot.driveController.getX(Hand.kLeft));
+            }
+
+            @Override
+            protected boolean isFinished() {
+                // Make this return true when this Command no longer needs to run execute()
+                return false;
+            }
+
+            @Override
+            protected void end() {
+                // Called once after isFinished returns true
+            }
+        };
+    }
+
+    public Command goXDistance(double distance) {
+        // The command is named "Sample Command" and requires this subsystem.
+        return new Command("GoXDistance", this) {
+            @Override
+            protected void initialize() {
+                gyroDrivePID.reset();
+                gyroDrivePID.setSetpoint(gyro.getAngle());
+                gyroDrivePID.enable();
+                leftEncoder.reset();
+                rightEncoder.reset();
+            }
+
+            @Override
+            protected void execute() {
+                drive.arcadeDrive(sandstormSpeed, gyroCorrection);
+            }
+
+            @Override
+            protected boolean isFinished() {
+                // Make this return true when this Command no longer needs to run execute()
+                if (leftEncoder.get() + rightEncoder.get() / 2 > distance)
```

Comment:
Remove comment, replace entire function with `return (leftEncoder.get() + rightEncoder.get()) / 2 > distance;`

Reply:
You need parenthesis when getting the average